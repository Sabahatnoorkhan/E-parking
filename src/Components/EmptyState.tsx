import React from 'react';
import { Box, Typography } from '@mui/material';
import RemoveCircleOutlineIcon from '@mui/icons-material/RemoveCircleOutline';

const EmptyState = () => {
  return (
    <Box
      sx={{
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
        justifyContent: 'center',
        height: '100vh',
        textAlign: 'center',
      }}
    >
       <RemoveCircleOutlineIcon
        sx={{
          fontSize: 100,
          color: 'action.disabled',
          marginBottom: '20px',
        }}
      />
      <Typography variant="h6" color="textSecondary">
        No records are Available !!
      </Typography>
    </Box>
  );
};

export default EmptyState;
