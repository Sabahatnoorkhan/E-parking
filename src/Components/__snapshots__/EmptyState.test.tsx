import React from 'react';
import { render } from '@testing-library/react';
import EmptyState from '../EmptyState';

describe('EmptyState Component', () => {
  it('Should match the snapshot', () => {
    const { asFragment } = render(<EmptyState />);
    expect(asFragment()).toMatchSnapshot();
  });
});