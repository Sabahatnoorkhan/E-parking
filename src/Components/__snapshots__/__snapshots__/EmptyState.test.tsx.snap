// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState Component Should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1fqmik4"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-3nkws3-MuiSvgIcon-root"
      data-testid="RemoveCircleOutlineIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 11v2h10v-2zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8"
      />
    </svg>
    <h6
      class="MuiTypography-root MuiTypography-h6 css-1jgmn5n-MuiTypography-root"
    >
      No records are Available !!
    </h6>
  </div>
</DocumentFragment>
`;

exports[`EmptyState Component should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1fqmik4"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-3nkws3-MuiSvgIcon-root"
      data-testid="RemoveCircleOutlineIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 11v2h10v-2zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8"
      />
    </svg>
    <h6
      class="MuiTypography-root MuiTypography-h6 css-1jgmn5n-MuiTypography-root"
    >
      No records are Available !!
    </h6>
  </div>
</DocumentFragment>
`;
